fabric/src/test/java/TestRegex.java

```diff
@@ -216,6 +216,10 @@ public void ChatRedirectFeature_LoginRedirector_FOREGROUND_PATTERN() {
         p.shouldMatch("\uE023 §#44aa3300v8j§a has just logged in!");
         // champion nickname
         p.shouldMatch("\uE017 §#ffe60000§ocharlie268IsAWizard§6 has just logged in!");
+        // moderator nickname, new style
+        p.shouldMatch("\uE01F §6§o§<1>Navi§e has just logged in!");
```

Comment:
Why are we using INCLUDE_EVENTS here? Does the method even use that for getting the string? (If so , why?)

Reply:
wdym?

Reply:
&<1> is an event (clickable or hoverable, I don't remember). I am asking why this method is matching clickable events, as we never do this, and only use regular MC-styles.